student/index.js

```diff
@@ -24,27 +29,33 @@ function shallowCopyObject (object) {
 // return a shallow copy of the array
 function shallowCopyArray (array) {
   // your code here
+  const NewArray=[...array];
+  return NewArray;
 }
 
 // Exercise 4
 // Given two objects,
 // turn them into one object that contains the properties of both objects
 function mergeTwoObjects (object1, object2) {
   // your code here
+  const NewMergedObject={...object1, ...object2};
+  return NewMergedObject;
 }
 
 // Exercise 5
 // Given two arrays,
 // turn them into one array containing all elements of both arrays
 function combineTwoArrays (array1, array2) {
   // your code here
+  const NewCombinedArray=[...array1,...array2];
+  return NewCombinedArray;
 }
-
+/*
 module.exports = {
   returnTheString,
   splitTheString,
   shallowCopyObject,
   shallowCopyArray,  
   mergeTwoObjects,
   combineTwoArrays,
-};
\ No newline at end of file
+};*/
```

Comment:
Your solutions are correct but the test is failing because you commented out this line and line 53 :) if you activate them again, the test will pass as well :)